update-imported-docs.sh

```diff
@@ -1,44 +1,38 @@
 #!/bin/bash
 
-set -x
+# Uncomment this to see the commands as they are run
+# set -x
 
 VERSION=1.3
 
-git clone --depth=1 https://github.com/kubernetes/kubernetes.git k8s
+#git clone --depth=1 -b release-$VERSION https://github.com/kubernetes/kubernetes.git k8s
```

Comment:
I'm not a huge fan of commenting old code rather than delete. Is this supposed to mean something specific?